enzyme/test/Enzyme/ReverseMode/allocnotape.ll

```diff
@@ -95,7 +95,7 @@ attributes #4 = { nounwind }
 ; CHECK-NEXT:   %1 = trunc i64 %iv to i32
 ; CHECK-NEXT:   %mul = mul nsw i32 %1, %1
 ; CHECK-NEXT:   %arrayidx = getelementptr inbounds [20 x i32], [20 x i32]* %0, i64 0, i64 %iv
-; CHECK-NEXT:   store i32 %mul, i32* %arrayidx, align 4, !tbaa !2
+; CHECK-NEXT:   store i32 %mul, i32* %arrayidx, align 4
```

Comment:
Can you keep the tbaa / invariant group here [except with a different or arbitrary id]